src/js/pensions/config/form.js

```diff
@@ -113,6 +114,25 @@ function usingDirectDeposit(formData) {
   return formData['view:noDirectDeposit'] !== true;
 }
 
+function servedDuringWartime(period) {
```

Comment:
I think we should pull this into the helper file and write some unit tests against it, rather than only doing the DefinitionTester test.